contracts/L1/gateway/L1Migrator.sol

```diff
@@ -115,7 +115,7 @@ contract L1Migrator is L1ArbitrumMessenger, IMigrator, EIP712 {
         uint256 _maxGas,
         uint256 _gasPriceBid,
         uint256 _maxSubmissionCost
-    ) external {
+    ) external payable {
```

Comment:
Let's add tests for the functions with the payable keyword. We can check assert that when msg.value is non-zero the function call succeeds - otherwise, without the payable keyword, the function call with a non-zero msg.value should fail.